ssh/src/main/java/ch/cyberduck/core/sftp/SFTPSession.java

```diff
@@ -286,11 +286,10 @@ private void authenticate(final SSHClient client, final Host host, final LoginCa
                 switch(Factory.Platform.getDefault()) {
                     case windows:
                         defaultMethods.add(new SFTPAgentAuthentication(client, new PageantAuthenticator()));
-                        break;
+                        // Break through
                     default:
                         defaultMethods.add(new SFTPAgentAuthentication(client, new OpenSSHAgentAuthenticator(
```

Comment:
Yeah, no.

The OpenSSHAgentAuthenticator won't work as a named pipe (`\.\pipe\openssh-ssh-agent`) is used instead of a Unix domain socket.[^1]

[^1]: https://github.com/PowerShell/Win32-OpenSSH/issues/1136#issuecomment-500549297

Reply:
See: https://github.com/PowerShell/openssh-portable/blob/latestw_all/contrib/win32/win32compat/ssh-agent/agent.c#L44

Don't know how that could be incorporated into sshj.

Reply:
Tracking: https://github.com/ymnk/jsch-agent-proxy/issues/34